Automattic/wc-calypso-bridge
# Remove store guided tour files

Fixes #426.

Removes store guided tour class and fixes PHPCS errors in the modified files.

It looks like GitHub doesn't recognize a file that was renamed, so it might be easier for you to review each commit for itself instead of reviewing the entire diff of this PR.

**Steps to test**

- Verify the plugin still loads without errors.
- Run tests and verify they still pass.
- Verify there are no regressions.
